src/scripts/plugincatalogpage.js

```diff
@@ -41,82 +41,31 @@ define(["loading", "libraryMenu", "globalize", "cardStyle", "emby-linkbutton", "
                 if (!options.targetSystem || plugin.targetSystem == options.targetSystem) {
                     return "UserInstalled" == plugin.type;
                 }
-
-                return false;
             }
-
             return false;
         });
-        availablePlugins = allPlugins.sort(function (a__e, b__r) {
-            var aName = a__e.category;
-            var bName = b__r.category;
 
-            if (aName > bName) {
+        availablePlugins = allPlugins.sort(function (a, b) {
+            if (a.category > b.category) {
                 return 1;
-            }
-
-            if (bName > aName) {
+            } else if (b.category > a.category) {
                 return -1;
             }
-
-            aName = a__e.name;
-            bName = b__r.name;
-
-            if (aName > bName) {
+            if (a.name > b.name) {
                 return 1;
-            }
-
-            if (bName > aName) {
+            } else if (b.name > a.name) {
                 return -1;
             }
-
             return 0;
         });
+
         var i__q;
```

Comment:
pls give this variable a more descriptive name... or if it is really an index variable rename it to just `i`